typespec-tests/src/main/java/com/cadl/enumservice/EnumServiceAsyncClient.java

```diff
@@ -303,7 +303,7 @@ public Mono<Response<BinaryData>> setIntEnumArrayWithResponse(
      * String
      * }</pre>
      *
-     * @param stringArray Array of Response.
+     * @param stringArray Array of SetStringEnumArrayResponse.
```

Comment:
These strange thing is done by `prenamer` which we copied from autorest modules.

So far, no service actually uses this option, so we may eventually remove this from typespec-java.